bundle/manifests/falcon.crowdstrike.com_falconcontainers.yaml

```diff
@@ -123,6 +123,9 @@ spec:
               phase:
                 description: Phase or the status of the deployment
                 type: string
+              retry_attempt:
+                description: 'RetryAttempt is number of previous failed attempts. Valid values: 0-5'
```

Comment:
Should we really limit retry attempts? Seems like we should set a default, and let whatever is desired for retry attempts to be set by the user.

Reply:
At this point we don't have a good way to distinguish recoverable errors from non-recoverable ones.

That leads me to think that unbounded retry may not be desirable.

What I can do is to remove the retry_count=5 cap, but then I will have to add ever growing delays.

Reply:
Alright. Guess we can make the upper limit configurable at a later date if needed.